challenges/primes.js

```diff
@@ -10,3 +10,25 @@
 */
 
 // YOUR CODE HERE
+
+//1. 
+
+let isPrime = function(num) {
+  if (num === 2 || num % 2 === 1) {
+    console.log(true);
```

Comment:
Again, here it would be better to `return` this answer rather than `console.log`ing it.